packages/labs/analyzer/src/lib/javascript/functions.ts

```diff
@@ -156,10 +157,13 @@ const getReturn = (
     .getTypeChecker()
     .getSignatureFromDeclaration(node);
   if (signature === undefined) {
-    throw new DiagnosticsError(
-      node,
-      `Could not get signature to determine return type`
+    analyzer.diagnostics.push(
+      createDiagnostic({
+        node,
+        message: `Could not get signature to determine return type`,
```

Comment:
If you know: Is this one actionable in any way? It'd be nice to comment here when this is possible if semi-normal source can produce it, or comment that it's unexpected. Maybe TODO?